plugins/inventory/hrobot.py

```diff
@@ -0,0 +1,180 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Ansible, but is an independent component.
+# This particular file snippet, and this file snippet only, is BSD licensed.
+# Modules you write using this snippet, which is embedded dynamically by Ansible
+# still belong to the author of the module, and may assign their own license
+# to the complete work.
+#
+# Copyright (c), Felix Fontein <felix@fontein.de>, 2019
+#
+# Simplified BSD License (see licenses/simplified_bsd.txt or https://opensource.org/licenses/BSD-2-Clause)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = r"""
+    name: hrobot
+    plugin_type: inventory
+    author:
+        - Oleksandr Stepanov
+    short_description: Hetzner Robot inventory source
+    requirements:
+        - python >= 3.9
+    description:
+        - Reads Server from Hetzner Robot API.
+        - Uses a YAML configuration file that ends with hrobot.(yml|yaml).
+    extends_documentation_fragment:
+        - constructed
+        - inventory_cache
```

Comment:
```suggestion
        - ansible.builtin.constructed
        - ansible.builtin.inventory_cache
        - community.hrobot.robot
```
I've added the `robot` docs fragment since this plugin should provide the same interface as all other content in this collection.